src/app/app.component.ts

```diff
@@ -1,12 +1,32 @@
-import { Component } from '@angular/core';
-import { RouterOutlet } from '@angular/router';
+import {Component, inject, OnInit} from '@angular/core';
+import { RouterOutlet, RouterModule } from '@angular/router';
+import { authService} from '@auth/services/auth.sevrice';
+import { NgIf } from '@angular/common';
+import { HeaderComponent } from '@layout/header/header.component'
+import {AuthStore} from '@auth/store/auth.store';
 
 @Component({
   selector: 'app-root',
-  imports: [RouterOutlet],
+  imports: [RouterOutlet, RouterModule, NgIf, HeaderComponent],
   templateUrl: './app.component.html',
   styleUrl: './app.component.css'
 })
-export class AppComponent {
-  title = 'angular-auth';
+export class AppComponent implements OnInit {
```

Comment:
suggestion: please take a look on best practice for authorization. In my opinion would be nice to move the things out of AppComponent, maybe some app initializer, maybe some auth module that will contain logic, please check best practice